Tasks/AzureIoTEdgeV2/deployimage.ts

```diff
@@ -131,35 +133,67 @@ class azureclitask {
     }
   }
 
-  static loginAzure() {
+  static async loginAzure(): Promise<void> {
```

Comment:
This pattern occurs across multiple tasks (e.g. AzureCLI, AzureContainerApps). Would not it be more clean to extract it into single shared helper method or some class?

Reply:
Ideally, yes, but there is no common package that has helpers for az CLI interaction

Reply:
Done - extracted common code to common artifacts library